app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -1000,6 +1001,16 @@ interface PledgeFragmentViewModel {
                     }
         }
 
+        private fun initialCardSelection(storedCards: List<StoredCard>, project: Project): Pair<StoredCard, Int>? {
+            val defaultIndex = 0
+            val backingPaymentSourceIndex = storedCards.indexOfFirst { it.id() == project.backing()?.paymentSource()?.id() }
+            return when {
+                backingPaymentSourceIndex != -1 -> Pair(storedCards[backingPaymentSourceIndex], backingPaymentSourceIndex)
```

Comment:
Is the `backingPaymentSourceIndex` set to -1 if `indexOfFirst ` doesn't find something?

Reply:
Yeah, that's what `indexOfFirst` returns https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/index-of-first.html